code/modules/projectiles/guns/hoboguns.dm

```diff
@@ -3,37 +3,59 @@
 
 
 /obj/item/gun/ballistic/automatic/hobo
-	slowdown = GUN_SLOWDOWN_PISTOL_LIGHT
+	slowdown = 0.3
```

Comment:
This and other slowdowns will need to be converted into a define, you can find those in code\__DEFINES\combat.dm

Reply:
ah shit, due to my stupid insistence on dragging and dropping files to github (instead of just using github desktop), I accidentally forgot to use our file for hoboguns.dm (and just copy my changes over)

Gimme a second, I'll fix that